src/components/bookmarkScreen/components/bookmarkItem.js

```diff
@@ -1,20 +1,24 @@
-import React from "react";
-import { Text, View, StyleSheet, FlatList, TouchableOpacity } from "react-native";
+import React, { useContext, useState } from "react";
+import { Text, View, StyleSheet, TouchableOpacity } from "react-native";
+import { BookmarksContext } from "../../../utils/bookmarks";
+import { useApi } from "../../../utils/api";
+
+export const BookmarkItem = ({bookmark, setDeletingStatus}) => {
+  const [bookmarks, setBookmarks] = useContext(BookmarksContext);
+  const api = useApi();
+
+  const deleteBookmark = () => {
+    setBookmarks(bookmarks.filter((bookmarkItem) => bookmark.item.question !== bookmarkItem.question));
+    api.deleteBookmark(bookmark.item.question).then((status) => setDeletingStatus(status)).catch((err) => console.log('deleteBookmark error', err))
```

Comment:
why - because you first set new state for you component and only after that you make a request. But what if request failed? In this case user will think that item was deleted, when it wasn't